.flake8

```diff
@@ -1,9 +1,7 @@
 [flake8]
 exclude =
     __init__.py,
-    versioneer.py,
-    _version.py,
-    dask_gateway_config.py
```

Comment:
- versioneer.py isn't used in this project
- dask_gateway_config.py may be generated, but is .gitignored as part of #499